main.go

```diff
@@ -411,16 +433,18 @@ func main() {
 	}
 
 	if err = (&controllers.ServingRuntimeReconciler{
-		Client:              mgr.GetClient(),
-		Log:                 ctrl.Log.WithName("controllers").WithName("ServingRuntime"),
-		Scheme:              mgr.GetScheme(),
-		ConfigProvider:      cp,
-		ConfigMapName:       types.NamespacedName{Namespace: ControllerNamespace, Name: UserConfigMapName},
-		ControllerNamespace: ControllerNamespace,
-		ControllerName:      controllerDeploymentName,
-		ClusterScope:        clusterScopeMode,
-		EnableCSRWatch:      enableCSRWatch,
-		RegistryMap:         registryMap,
+		Client:                mgr.GetClient(),
+		Log:                   ctrl.Log.WithName("controllers").WithName("ServingRuntime"),
+		Scheme:                mgr.GetScheme(),
+		ConfigProvider:        cp,
+		ConfigMapName:         types.NamespacedName{Namespace: ControllerNamespace, Name: UserConfigMapName},
+		ControllerNamespace:   ControllerNamespace,
+		ControllerName:        controllerDeploymentName,
+		ClusterScope:          clusterScopeMode,
+		EnableCSRWatch:        enableCSRWatch,
+		EnableSecretWatch:     enableSecretWatch,
+		DeployPVCForPredictor: conf.EnableDeployPVCForPredictor,
```

Comment:
Getting the value here means it won't be dynamic. We can just grab it from the config where we use it.